### PR TITLE
Restructure of the pgfexamples

### LINKED ISSUE #42: Restructure `pgfexamples`
Remove duplicates and sort by type

### PR DESCRIPTION
**Pull request by dominikgeissler**
#### Changed
* New structure of the examples based on their type (closes #42)

#### Removed
* Non-working / duplicate examples

### CODE REVIEW

File: pgfexamples/inference/telephone_operator_param.pgcl
Comment (LKlinke) [inline pgfexamples/inference/telephone_operator_param.pgcl:1]: why is this deleted and not moved?

File: pgfexamples/equivalence/geometric_invariant.pgcl
Comment (LKlinke) [inline pgfexamples/equivalence/geometric_invariant.pgcl:1]: I can't find this in the new structure

File: pgfexamples/equivalence/geometric_parameter.pgcl
Comment (LKlinke) [inline pgfexamples/equivalence/geometric_parameter.pgcl:1]: ****

File: pgfexamples/equivalence/geometric_parameter_invariant.pgcl
Comment (LKlinke) [inline pgfexamples/equivalence/geometric_parameter_invariant.pgcl:1]: I can't find this in the new structure

File: pgfexamples/equivalence/random_walk_invariant.pgcl
Comment (LKlinke) [inline pgfexamples/equivalence/random_walk_invariant.pgcl:1]: I can't find this in the new structure

File: pgfexamples/equivalence/random_walk_parameter_invariant.pgcl
Comment (LKlinke) [inline pgfexamples/equivalence/random_walk_parameter_invariant.pgcl:1]: I can't find this in the new structure

File: pgfexamples/EVTInvariants/run_benchmarks.sh
Comment (LKlinke) [inline pgfexamples/EVTInvariants/run_benchmarks.sh:1]: is this not needed anymore because you have a new benchmark script?

**Review by LKlinke — CHANGES_REQUESTED** [review]
please explain the missing files.

File: pgfexamples/equivalence/geometric_parameter.pgcl
Comment (dominikgeissler) [reply pgfexamples/equivalence/geometric_parameter.pgcl:1]: under [`https://github.com/LKlinke/Prodigy/blob/42/restructure-examples/pgfexamples/template_parameter_synthesis/geometric_parameter.pgcl`](https://github.com/LKlinke/Prodigy/blob/42/restructure-examples/pgfexamples/template_parameter_synthesis/geometric_parameter.pgcl)

should this be in `loop_equivalence` as well?

File: pgfexamples/equivalence/geometric_parameter_invariant.pgcl
Comment (dominikgeissler) [reply pgfexamples/equivalence/geometric_parameter_invariant.pgcl:1]: under [`pgfexamples/template_parameter_synthesis/invariants/geometric_parameter_invariant.pgcl`](https://github.com/LKlinke/Prodigy/blob/42/restructure-examples/pgfexamples/template_parameter_synthesis/invariants/geometric_parameter_invariant.pgcl)

File: pgfexamples/equivalence/random_walk_invariant.pgcl
Comment (dominikgeissler) [reply pgfexamples/equivalence/random_walk_invariant.pgcl:1]: under [`pgfexamples/loop_equivalence/invariants/random_walk_invariant.pgcl`](https://github.com/LKlinke/Prodigy/blob/42/restructure-examples/pgfexamples/loop_equivalence/invariants/random_walk_invariant.pgcl)

File: pgfexamples/EVTInvariants/run_benchmarks.sh
Comment (dominikgeissler) [reply pgfexamples/EVTInvariants/run_benchmarks.sh:1]: yes exactly

File: pgfexamples/equivalence/random_walk_parameter_invariant.pgcl
Comment (dominikgeissler) [reply pgfexamples/equivalence/random_walk_parameter_invariant.pgcl:1]: under [`pgfexamples/template_parameter_synthesis/invariants/random_walk_parameter_invariant.pgcl`](https://github.com/LKlinke/Prodigy/blob/42/restructure-examples/pgfexamples/template_parameter_synthesis/invariants/random_walk_parameter_invariant.pgcl)

File: pgfexamples/inference/telephone_operator_param.pgcl
Comment (dominikgeissler) [reply pgfexamples/inference/telephone_operator_param.pgcl:1]: i've added it again under [`pgfexamples/template_parameter_synthesis/telephone_operator_parameter.pgcl`](https://github.com/LKlinke/Prodigy/blob/42/restructure-examples/pgfexamples/template_parameter_synthesis/telephone_operator_parameter.pgcl)

**Review by LKlinke — APPROVED** [review]
